chef/cookbooks/provisioner/recipes/base.rb

```diff
@@ -301,6 +301,13 @@
         %x{zypper --non-interactive addrepo --refresh #{attrs[:url]} #{name}}
       end
     end
+    # install additional packages
+    os = "#{node[:platform]}-#{node[:platform_version]}"
+    packages = [ ]
+    if node[:provisioner][:packages][os]
+      packages = node[:provisioner][:packages][os]
+    end
+    system("zypper --non-interactive install #{packages.join(' ')}")
```

Comment:
No, good point. You mean like this:
```ruby
    if node[:provisioner][:packages][os]
      node[:provisioner][:packages][os].each do |p|
        package p do
          action :install
        end
      end
    end
```
This also has the advantage that `base.rb` throws an error if one of the packages couldn't be installed. With my current approach, `zypper` does not really complain about missing packages.